website/docs/api/themes/theme-configuration.mdx

```diff
@@ -794,7 +794,7 @@ const defaultMagicComments = [
 
 ### Theme {#theme}
 
-By default, we use [Palenight](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/themes/palenight.js) as syntax highlighting theme. You can specify a custom theme from the [list of available themes](https://github.com/FormidableLabs/prism-react-renderer/tree/master/src/themes). You may also use a different syntax highlighting theme when the site is in dark mode.
+By default, we use [Palenight](https://github.com/FormidableLabs/prism-react-renderer/blob/bc79d72268405434d895e19c15e4a484bf1d1b2c/packages/prism-react-renderer/src/themes/palenight.ts) as syntax highlighting theme. You can specify a custom theme from the [list of available themes](https://github.com/FormidableLabs/prism-react-renderer/tree/bc79d72268405434d895e19c15e4a484bf1d1b2c/packages/prism-react-renderer/src/themes). You may also use a different syntax highlighting theme when the site is in dark mode.
```

Comment:
```suggestion
By default, we use [Palenight](https://github.com/FormidableLabs/prism-react-renderer/blob/master/packages/prism-react-renderer/src/themes/palenight.ts) as syntax highlighting theme. You can specify a custom theme from the [list of available themes](https://github.com/FormidableLabs/prism-react-renderer/tree/master/packages/prism-react-renderer/src/themes). You may also use a different syntax highlighting theme when the site is in dark mode.
```

thanks

let's use master for now, as we did for https://github.com/facebook/docusaurus/pull/8904

We'll look into setting up a link substitution in sync with the current prism version later